cmd/monitor.go

```diff
@@ -0,0 +1,245 @@
+// Copyright © 2019 The VirusTotal CLI authors. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/VirusTotal/vt-cli/utils"
+	vt "github.com/VirusTotal/vt-go"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var base64RegExp = `^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$`
+
+// NewMonitorItemsListCmd returns a list or monitor_items according to a filter.
+func NewMonitorItemsListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List monitor items in your account",
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			p, err := NewPrinter(cmd)
+			if err != nil {
+				return err
+			}
+			return p.PrintCollection(vt.URL("monitor/items"))
+		},
+	}
+
+	addIncludeExcludeFlags(cmd.Flags())
+	addFilterFlag(cmd.Flags(), "path:/")
```

Comment:
Why passing "path:/" by default as a filter? I would expect that the "list" command without any filters will return all the items in my root path, why the filter must be explicitly specified in this case?

Reply:
Right, API was not doing this by default, corrected in https://github.com/VirusTotal/vt-core/pull/1242